_includes/header.html

```diff
@@ -78,6 +78,11 @@
 <script src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=VA" id="_fed_an_ua_tag"></script>
 <script src="/assets/js/generated/bundle.js"></script>
 
+<!--[if lt IE 9]>
```

Comment:
This shouldn't be needed anymore I don't think.

Though I'm not sure how to test...